src/webgpu/shader/validation/parse/diagnostic.spec.ts

```diff
@@ -106,6 +106,25 @@ g.test('invalid_severity')
     t.expectCompileResult(false, code);
   });
 
+g.test('duplicate_attribute')
```

Comment:
Should this just be a modification to the `conflicting_attribute_same_location` test below? That one allows conflicts when the severity is the same (as per Tint's current behavior), but as discussed offline we want to change that.